test/functional/mempool_sigoplimit.py

```diff
@@ -133,6 +136,49 @@ def test_sigops_limit(self, bytes_per_sigop, num_sigops):
         assert_equal(entry_parent['descendantcount'], 2)
         assert_equal(entry_parent['descendantsize'], parent_tx.get_vsize() + sigop_equivalent_vsize)
 
+    def test_sigops_package(self):
+        # Make a 2-transaction package which fails vbyte checks even though
+        # separately they would work.
+        self.restart_node(0, extra_args=["-bytespersigop=5000"] + self.extra_args[0])
+
+        _, pubkey = generate_keypair()
+        amount_for_bare = 50000
+        tx_parent_dict = self.wallet.create_self_transfer(fee=Decimal("3"))
+        tx_parent_utxo = tx_parent_dict["new_utxo"]
+        tx_parent = tx_parent_dict["tx"]
+        tx_parent.vout.append(CTxOut(amount_for_bare, keys_to_multisig_script([pubkey], k=1)))
+        tx_parent.vout[0].nValue -= amount_for_bare
+        tx_parent.calc_sha256()
+        tx_parent_utxo["txid"] = tx_parent.hash
+        tx_parent_utxo["value"] -= Decimal("0.00005000")
+
+        tx_child_dict = self.wallet.create_self_transfer(utxo_to_spend=tx_parent_utxo, fee=Decimal("3"))
+        tx_child_utxo = tx_child_dict["new_utxo"]
+        tx_child = tx_child_dict["tx"]
+        tx_child.vout.append(CTxOut(amount_for_bare, keys_to_multisig_script([pubkey], k=1)))
+        tx_child.vout[0].nValue -= amount_for_bare
+        tx_child.calc_sha256()
+        tx_child_utxo["txid"] = tx_child.hash
+        tx_child_utxo["value"] -= Decimal("0.00005000")
+
+        # Separately, the parent tx is ok
+        parent_individual_testres = self.nodes[0].testmempoolaccept([tx_parent.serialize().hex()])[0]
+        assert parent_individual_testres["allowed"]
+        # Multisig is counted as MAX_PUBKEYS_PER_MULTISIG = 20 sigops
+        assert_equal(parent_individual_testres["vsize"], 5000 * 20)
+
+        # But together, it's exceeding limits in the *package* context. If sigops adjusted vsize wasn't being checked
+        # here, it would get farther in validation and give too-long-mempool-chain error instead.
```

Comment:
nit 5c48501e9f1a3da5307b1bee08396598398311e8

```suggestion
        # here, it would get further in validation and give too-long-mempool-chain error instead.
```

Reply:
done